src/mpid/ch4/src/ch4r_rma_target_callbacks.c

```diff
@@ -20,7 +20,7 @@ static void win_post_proc(const MPIDIG_win_cntrl_msg_t * info, MPIR_Win * win);
 static void win_unlock_done(const MPIDIG_win_cntrl_msg_t * info, MPIR_Win * win);
 static int handle_acc_cmpl(MPIR_Request * rreq);
 static int handle_get_acc_cmpl(MPIR_Request * rreq);
-static void handle_acc_data(MPI_Aint in_data_sz, MPIR_Request * rreq);
+static void handle_acc_data(MPI_Aint msg_data_sz, MPIR_Request * rreq);
```

Comment:
Rather than passing the `msg_data_sz` which will copy it to `rreq` with `MPIDIG_recv_init` (or one of such routines), why not let caller call `MPIDIG_recv_init` directly and just pass `rreq` here?